tests/contrib/django/django_app/settings.py

```diff
@@ -50,11 +50,13 @@
         "BACKEND": "django.core.cache.backends.memcached.PyLibMCCache",
         "LOCATION": "127.0.0.1:11211",
     },
-    "python_memcached": {
+}
+
+if django.VERSION <= (4, 1, 0):
```

Comment:
I sort of expect (hope?) that some tests fail with this change, otherwise why is it here at all 🤔

Reply:
Do you mean if anything fails with this change, it means that we aren't using `python_memcached` hence why don't remove all this lines?

Reply:
yeah, that is my thinking.